src/index.html

```diff
@@ -11,7 +11,139 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="post"
+          onsubmit="onSubmit()"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <div>
+          <label for="sname">Surname:</label>
+          <input type="text"
+                 name="sname"
+                 id="sname"
+                 autocomplete="off"
+                 required
+          >
```

Comment:
```suggestion
          <input 
            type="text"
            name="sname"
            id="sname"
            autocomplete="off"
            required
          >
```
Форматируй теги таким образом